packages/graphback-codegen-resolvers/src/plugins/ResolverGeneratorPlugin.ts

```diff
@@ -0,0 +1,86 @@
+import { getModelTypesFromSchema, GraphbackCoreMetadata, GraphbackPlugin, TypeScriptResolversVisitor } from '@graphback/core';
+import { buildASTSchema, GraphQLField, GraphQLFieldMap, GraphQLObjectType, GraphQLOutputType, GraphQLSchema, isListType, isNonNullType, parse, printSchema, visit } from 'graphql';
+import { ResolverGeneratorOptions } from '../api/ResolverGeneratorOptions';
+import * as pluralize from "pluralize";
+
+export interface ResolverDefinition {
+    generated: ResolverBaseTypeMap[]
+}
+
+export interface ResolverBaseTypeMap {
+    [typeName: string]: ResolverMap
+}
+
+export interface ResolverMap {
+    Query?: ResolverFunctionMap[]
+}
+
+export interface ResolverFunctionMap {
+    [name: string]: string;
+}
+
+export class ResolverGeneratorPlugin extends GraphbackPlugin {
+    private resolverDefinition: ResolverDefinition;
+    constructor(config?: ResolverGeneratorOptions) {
+        super();
+        this.resolverDefinition = {
+            generated: []
+        }
+    }
+
+    public getPluginName() {
+        return 'RESOLVER_GENERATOR';
+    }
+
+    public transformSchema(metadata: GraphbackCoreMetadata): GraphQLSchema {
+        return metadata.getSchema();
+    }
+
+    public createResolvers(schema: GraphQLSchema) {
+        const models = getModelTypesFromSchema(schema);
+        const queryType = schema.getQueryType();
+        const queryFields = queryType.getFields();
+
+        for (const model of models) {
+            this.resolverDefinition[model.name] = {
```

Comment:
We do not base on query fields. This is not possible as we do not know what queryField is custom and what is not. We should base on CRUD config from metadata